nodejs/awsx/ec2/vpc.ts

```diff
@@ -447,6 +409,31 @@ export interface VpcSubnetArgs {
     tags?: pulumi.Input<aws.Tags>;
 }
 
+/**
+ * Alias for a cidr block.
+ */
+export type CidrBlock = string;
+
+export interface VpcSubnetLocation {
+    /**
+     * The AZ for the subnet.
+     */
+    availabilityZone?: string;
+    /**
+     * The AZ ID of the subnet.
+     */
+    availabilityZoneId?: string;
+    /**
+     * The CIDR block for the subnet.
+     */
+    cidrBlock: pulumi.Input<CidrBlock>;
+    /**
+     * The IPv6 network range for the subnet, in CIDR notation. The subnet size must use a /64
+     * prefix length.
+     */
+    ipv6CidrBlock?: pulumi.Input<string>;
```

Comment:
Should this be `pulumi.Input<CidrBlock>`?

Reply:
I think we may want to define it differently.  i.e. something like Ipv6CidrBlock.  The reason being that these cidr addresses have very different formats, and i'd like to distinguish them (even if they're just strings).